docs/dapp/sapphire/precompiles.md

```diff
@@ -140,8 +140,16 @@ ed25519-based are equivalent, and all secp256k1-based methods are
 equivalent. `Sr25519` is not available and will return an error.
 
 ### Gas Cost
-* `0` (`Ed25519Oasis`), `1` (`Ed25519Pure`), `2` (`Ed25519PrehashedSha512`) - ed25519: cost 35,000,
-* `3` (`Secp256k1Oasis`), `4` (`Secp256k1PrehashedKeccak256`), `5` (`Secp256k1PrehashedSha256`) - secp256k1: cost 110,000 gas.
+* Ed25519: 1,000 gas
```

Comment:
(nit) I wonder if we should use commas.

Reply:
I just followed the convention in the existing document.